File: src/com/fenghuo/notes/FragmentExt.java
package com.fenghuo.notes;

import android.support.v4.app.Fragment;
import android.view.KeyEvent;

/**
 * @author Administrator 添加事件接口的FragMent
 */
public class FragmentExt extends Fragment implements KeyEvent.Callback {

    @Override
    public boolean onKeyDown(int keyCode, KeyEvent event) {
        return false;
    }

    @Override
    public boolean onKeyLongPress(int keyCode, KeyEvent event) {
        return false;
    }

    @Override
    public boolean onKeyUp(int keyCode, KeyEvent event) {
        return false;
    }

    @Override
    public boolean onKeyMultiple(int keyCode, int count, KeyEvent event) {
        return false;
    }
}

File: src/com/fenghuo/notes/FragmentExt.kt
package com.fenghuo.notes

import android.support.v4.app.Fragment
import android.view.KeyEvent

/**
 * @author Administrator 添加事件接口的FragMent
 */
open class FragmentExt : Fragment(), KeyEvent.Callback {

    override fun onKeyDown(keyCode: Int, event: KeyEvent): Boolean {
        return false
    }

    override fun onKeyLongPress(keyCode: Int, event: KeyEvent): Boolean {
        return false
    }

    override fun onKeyUp(keyCode: Int, event: KeyEvent): Boolean {
        return false
    }

    override fun onKeyMultiple(keyCode: Int, count: Int, event: KeyEvent): Boolean {
        return false
    }
}
